### PR TITLE
Change header in PL fingerprint

### PR DESCRIPTION
**Pull request by mimi89999**

### CODE REVIEW

**Comment by mimi89999** [comment]
Pipeline failing is unrelated:
```
/usr/bin/git push --quiet --force ***github.com/ooni/pipeline.git gh-pages
remote: Permission to ooni/pipeline.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/ooni/pipeline.git/': The requested URL returned error: 403
Error: The process '/usr/bin/git' failed with exit code 128
```

File: af/fastpath/fastpath/utils.py [hunk 1]
```diff
@@ -430,9 +430,9 @@
     ],
     "PL": [
         # https://github.com/ooni/pipeline/issues/224
-        # https://explorer.ooni.org/measurement/20190911T110527Z_AS6830_kkm1ZGUCJI4dSrRV4xl6QwG77o7EeI2PDKbwt9SPL9BBJHUsTr?input=http://www.eurogrand.com/
+        # https://explorer.ooni.org/measurement/20210409T092428Z_webconnectivity_PL_12912_n1_DoMtAEf0oFS4J7oc?input=http://www.eurogrand.com/
         {
-            "header_full": "http://www.finanse.mf.gov.pl/inne-podatki/podatek-od-gier-gry-hazardowe/komunikat",
+            "header_full": "https://www.finanse.mf.gov.pl/inne-podatki/podatek-od-gier-gry-hazardowe/komunikat",
```
Comment (hellais) [inline af/fastpath/fastpath/utils.py:441]: I think we want to keep both of the fingerprints so that we can also mark the older measurements as confirmed

**Comment by hellais** [comment]
LGTM

**Review by hellais — APPROVED** [review]
Thanks for working on this! It looks good to merge.

**Pull request merged**